### ISSUE TITLE
natvis support for GSL

### ISSUE THREAD

**Issue by LYP951018** [Microsoft/GSL#430]
For reference:
``` XML
<?xml version="1.0" encoding="utf-8"?>  
<AutoVisualizer xmlns="http://schemas.microsoft.com/vstudio/debugger/natvis/2010">  
 <Type Name="gsl::span&lt;*,-1&gt;">
    <Intrinsic Name="size" Expression="storage_.size_" />
    <DisplayString>{{ size={size()} }}</DisplayString>
    <Expand>
      <ArrayItems>
        <Size>size()</Size>
        <ValuePointer>storage_.data_</ValuePointer>
        </ArrayItems>
    </Expand>
  </Type>

  <Type Name="gsl::span&lt;*,*&gt;">
    <Intrinsic Name="size" Expression="$T2" />
    <DisplayString>{{ size={size()} }}</DisplayString>
    <Expand>
      <ArrayItems>
        <Size>size()</Size>
        <ValuePointer>storage_.data_</ValuePointer>
        </ArrayItems>
    </Expand>
  </Type>
</AutoVisualizer>
```
Hope that helps :)

**Comment by neilmacintosh** [comment]
Thanks @LYP951018, I'm sure that Visual Studio users will find that particularly helpful.

**Comment by neilmacintosh** [comment]
@LYP951018 I'd be happy to add this to the repo as a file, so that others can easily consume it. Would you be able to put together a PR that adds the file? That way I can be sure we've got all the right legal sign-off to accept the contribution.

**Comment by neilmacintosh** [comment]
This issue has now been closed by #440. Thanks for the original suggestion @LYP951018  and thanks for the PR @GuildMasterInfinite.
